### PR TITLE
Set public and private IP addresses

### PR DESCRIPTION
**Pull request by edevenport**
### What does this PR do?

This PR allows a user to set reserved public IP addresses and private IP addresses.

### Previous Behavior

Previously only dynamically assigned IP addresses were allowed.

### New Behavior

Dynamically assigned IP addresses are still the default, but reserved "status" IP addresses can now be assigned.

### Tests written?

No

### Commits signed with GPG?

No

### CODE REVIEW

File: doc/topics/cloud/profitbricks.rst [hunk 1]
```diff
@@ -199,6 +203,9 @@ public_lan
     LAN exists, then a new public LAN will be created. The value accepts a LAN
     ID (integer).
 
+public_ips
+    Public IPs assigned to the NIC in the public LAN.
+
```
Comment (rallytime) [inline doc/topics/cloud/profitbricks.rst]: Adding a `.. versionadded:: Fluorine` directive would be good for this new option.

File: doc/topics/cloud/profitbricks.rst [hunk 2]
```diff
@@ -218,6 +225,9 @@ private_lan
     LAN exists, then a new private LAN will be created. The value accepts a LAN
     ID (integer).
 
+private_ips
+    Private IPs assigned in the private LAN. NAT setting is ignored when this setting is active.
+
```
Comment (rallytime) [inline doc/topics/cloud/profitbricks.rst]: Same comment here as above.

**Review by rallytime — CHANGES_REQUESTED** [review]
Just one small change requested, otherwise, this looks good.

**Comment by denza** [comment]
@rallytime A version tag added.

**Comment by denza** [comment]
@rallytime Could you please let us know if you require any further changes.

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
@denza Looks good! Thanks!

**Pull request merged**